src/coreclr/jit/instrsxarch.h

```diff
@@ -621,6 +621,17 @@ INST3(lzcnt,            "lzcnt",            IUM_WR, BAD_CODE,     BAD_CODE,
 // POPCNT
 INST3(popcnt,           "popcnt",           IUM_WR, BAD_CODE,     BAD_CODE,     SSEFLT(0xB8),                            Resets_OF      | Resets_SF     | Writes_ZF     | Resets_AF     | Resets_PF     | Resets_CF )
 
+// SHA1
+INST3(sha1rnds4,        "sha1rnds4",       IUM_WR, BAD_CODE,     BAD_CODE,  0xcc,                                        INS_FLAGS_None )
+INST2(sha1nexte,        "sha1nexte",       IUM_WR, BAD_CODE,     0xc8,      INS_FLAGS_None )
+INST2(sha1msg1,         "sha1msg1",        IUM_WR, BAD_CODE,     0xc9,      INS_FLAGS_None )
+INST2(sha1msg2,         "sha1msg2",        IUM_WR, BAD_CODE,     0xca,      INS_FLAGS_None )
```

Comment:
These aren't correct. The manual indicates `NP 0F 3A **` and `NP 0F 38 **` so you need `SSE38(0x**)` and `SSE3A(0x**)`, respectively.

I believe they should all be `INST3` as well, with `IUM_WR, BAD_CODE, BAD_CODE, SSE3*(0x**)`. This is because thje ordering is:
```
//    id                nm                  um      mr            mi            rm                                       flags
```

and in all cases these are `rm` instructions (register first, memory second).